src/pip/_internal/vcs/bazaar.py

```diff
@@ -40,13 +39,12 @@ def export(self, location):
         if os.path.exists(location):
             rmtree(location)
 
-        with TempDirectory(kind="export") as temp_dir:
-            self.unpack(temp_dir.path)
-
-            self.run_command(
-                ['export', location],
-                cwd=temp_dir.path, show_stdout=False,
-            )
+        url, rev = self.get_url_rev()
+        rev_options = self.make_rev_options(rev)
+        self.run_command(
+            ['export', location, url] + rev_options,
```

Comment:
You also need to call ‘to_args()’ (like it’s being done in obtain).

Reply:
Done.